DH_Engine/Classes/DHSquadReplicationInfo.uc

```diff
@@ -1947,7 +2026,6 @@ simulated function RallyPointPlacementResult GetRallyPointPlacementResult(DHPlay
         return Result;
     }
 
-    GRI = DHGameReplicationInfo(PC.GameReplicationInfo);
```

Comment:
The global `GRI` is used in the simulated function here. It'll always be `none` on the client.

Reply:
This is sorted now.